assemblies/cde-renderer/feature/src/main/feature/feature.xml

```diff
@@ -10,7 +13,7 @@
   <feature name="pentaho-requirejs-osgi-manager" version="${pentaho-osgi-bundles.version}">
     <feature>http</feature>
 
-    <bundle>mvn:commons-lang/commons-lang/2.6</bundle>
+    <bundle>mvn:commons-lang/commons-lang/${commons-lang.version}</bundle>
     <bundle>wrap:mvn:com.github.zafarkhaja/java-semver/0.9.0</bundle>
```

Comment:
shouldn't all version values be declared using properties?

Reply:
it should be that way, but the feature "pentaho-requirejs-osgi-manager" is here temporarily and there is no need for CDE to know the "java-semver" version.